app/controllers/application_controller.rb

```diff
@@ -86,6 +86,8 @@ def render_forbidden(**kwargs)
 
     respond_to do |format|
       format.any {
+        response.content_type = Mime::Type.lookup_by_extension(:jsonapi)
```

Comment:
I think we should respond with a JSON content type if asked (or if the route uses the `:json` format), instead of always responding with JSONAPI.